static/app/views/projectInstall/createProject.tsx

```diff
@@ -285,8 +318,16 @@ function CreateProject() {
     submitTooltipText = t('Please provide a project name');
   } else if (isMissingAlertThreshold) {
     submitTooltipText = t('Please provide an alert threshold');
+  } else if (isMissingMessagingIntegrationChannel) {
+    submitTooltipText = t(
+      'Please provide an integration channel for alert notifications'
+    );
   }
 
+  const keyToErrorText = {
+    actions: t('Notify via integration'),
+  };
+
```

Comment:
I haven't been able to cause any other error messages, so right now the key --> error text mapping is very sparse 😔

Reply:
Can we trace back what they could be?